css/at-rules/font-feature-values.json

```diff
@@ -0,0 +1,77 @@
+{
+  "css": {
+    "at-rules": {
+      "font-feature-values": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-feature-values",
```

Comment:
We should add "description": "\<code>@font-feature-values\</code>" for at-rules.